2-Node/nodeTool.js

```diff
@@ -0,0 +1,112 @@
+require("dotenv").config();
+const { Octokit } = require("@octokit/core");
+
+const regExp = new RegExp(process.argv[2]);
+
+const userName = "juan-fwn";
+const repositoryName = "apprenticeship";
+const token = process.env.GITHUB_API_KEY;
+
+const getGitHubAPI = async (URL) => {
+  const octokit = new Octokit({
+    auth: token,
+  });
+
+  const response = await octokit.request(URL);
+
+  return response;
+};
+
+const getCardsInfo = async (URL) => {
+  const { data: cardInfo = [] } = await getGitHubAPI(URL);
+
+  return cardInfo.title;
+};
+
+const getCardsByColumn = async (URL) => {
+  const { data: cardsAPI = [] } = await getGitHubAPI(URL);
+
+  const cards = [];
+
+  for (const card of cardsAPI) {
+    const cardTitle = await getCardsInfo(card.content_url);
+    cards.push(cardTitle);
+  }
+
+  return cards;
+};
+
+const getColumnsByProyect = async (URL) => {
```

Comment:
It would be good if we update where it says proyect/s to project/s so the code is more consistent, I noticed we use both ways but the correct one is project/s